vowpalwabbit/vw.h

```diff
@@ -44,13 +44,16 @@ namespace VW
     (2) The code is not yet reentrant.
    */
 vw* initialize(std::unique_ptr<config::options_i, options_deleter_type> options, io_buf* model = nullptr,
-    bool skipModelLoad = false, trace_message_t trace_listener = nullptr, void* trace_context = nullptr);
+    bool skipModelLoad = false, trace_message_t trace_listener = nullptr, void* trace_context = nullptr,
```

Comment:
I do not think this should be added to the standard `initialize` calls if it is for testing. Can this be an internal only `initialize` implementation?

Reply:
I was planning on refactoring python reduction work to re-use this work, and it would use this same vw.h API. Thoughts?

Reply:
it could also be a test and binding only API (controlled via cmake)

Reply:
I see, I still think keeping it separate to these entrypoints may make things a little easier. Can it be called `initialize_custom` or something?

Reply:
done